lib/pliny/metrics.rb

```diff
@@ -0,0 +1,20 @@
+module Pliny
+  module Metrics
+    def count(*names, value: 1)
+      counts = Hash[names.map { |n| ["count##{Config.app_name}.#{n}", value] }]
+      Pliny.log(counts)
+    end
+
+    def measure(*names, &block)
+      start = Time.now
+      return_value = block.call
+      elapsed = Time.now - start
+
+      measures = Hash[names.map { |n| ["measure##{Config.app_name}.#{n}", elapsed] }]
```

Comment:
That might be over-extracting, but there could be another module for that method so we don't pollute the main class.

_fairly bad idea (or not): we could also prepend the method name with two underscores_